cookbook/core/intermediate/map_task.py

```diff
@@ -0,0 +1,44 @@
+"""
+Write a map task
+------------------------------------------
+
+A map task lets you run a task over a list of inputs within a single workflow node. This means you can run thousands
+of instances of that task without creating a node for every instance, providing valuable performance gains. Furthermore
+you can run map tasks on alternate execution back-ends, such as `AWS Batch <https://aws.amazon.com/batch/>`__ which is
+a provisioned services that can scale to great sizes.
```

Comment:
Do we have existing documentation on how to configure propeller to determine where to run an array task?

Reply:
I tried searching for batch as I was writing this comment and couldn't find anything